tests/roxctl/bats-tests/local/central-generate-psps.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
+
+load "../helpers.bash"
+
+out_dir=""
+
+setup_file() {
+    # remove binaries from the previous runs
+    [[ -n "$NO_BATS_ROXCTL_REBUILD" ]] || rm -f "${tmp_roxctl}"/roxctl*
+
+    echo "Testing roxctl version: '$(roxctl-release version)'" >&3
+    command -v yq >/dev/null || skip "Tests in this file require yq"
+}
+
+setup() {
+    export out_dir="$(mktemp -d -u)"
+}
+
+teardown() {
+    rm -rf "$out_dir"
+}
+
+@test "roxctl central generate k8s --enable-pod-security-policies=false should not output PSPs" {
+    roxctl central generate k8s pvc --output-dir="$out_dir" --enable-pod-security-policies=false
```

Comment:
Bats local tests in other files use either `roxctl-release` or `roxctl-development` instead of plain `roxctl`. The reason is long to explain but you can grok it pretty quickly if you check `helpers.bash`. In this test it wouldn't make a difference whether to use -release or -development so please choose either one you like.

```suggestion
    roxctl-release central generate k8s pvc --output-dir="$out_dir" --enable-pod-security-policies=false
```

Please update other tests similarly.